src/systems/wheel_slip/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+gz_add_system(wheel-slip
+  SOURCES
+  WheelSlip.cc
+  PUBLIC_LINK_LIBS
+    ignition-common${IGN_COMMON_VER}::ignition-common${IGN_COMMON_VER}
+    ignition-physics${IGN_PHYSICS_VER}::ignition-physics${IGN_PHYSICS_VER}
```

Comment:
I don't think this plugin uses any `ign-physics` APIs, so I think it's safe to remove it here.

Reply:
a1b6d997612965bd16cdd64dabb09585855e1bdf